TWLight/templates/header_partial_b4.html

```diff
@@ -11,11 +11,34 @@
           {% trans "The Wikipedia Library home" %}
         ">
   </a>
-  <a class="navbar-brand" href="{% url 'users:my_library' %}">
+  <a class="navbar-brand site-title-anchor" href="{% url 'users:my_library' %}">
     <!-- site name -->
     {% comment %}Translators: Name of the project/tool.{% endcomment %}
     <p class="site-title align-middle"> {% trans "The Wikipedia Library" %} </p>
   </a>
+  <!-- EBSCO Search Box Begins -->
+  <form action="https://searchbox.ebsco.com/search/" target="_blank" class="ebsco-single-search navbar-brand">
+    <input name="schemaId" value="search" type="hidden" />
+    <input name="custid" value="ns253359" type="hidden" />
+    <input name="groupid" value="main" type="hidden" />
+    <input name="profid" value="eds" type="hidden" />
+    <input name="proxy" value="https://wikipedialibrary.idm.oclc.org:9443/login?url=" type="hidden" />
+    <input name="scope" value="site" type="hidden" />
+    <input name="site" value="eds-live" type="hidden" />
+    <input name="direct" value="true" type="hidden" />
+    <div class="input-group eds-searchbox">
+      {% comment %}Translators: Shown in the EBSCO Discovery Search input. {% endcomment %}
```

Comment:
I'd probably leave "EBSCO Discovery" out of this comment, it's more likely to be confusing to translators I think.

Reply:
Done!